manifests/03-rbac.yaml

```diff
@@ -69,6 +69,22 @@ rules:
 
 ---
 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-samples-operator-proxy-reader
+rules:
+  - apiGroups:
+      - config.openshift.io
+    resources:
+      - proxies
+    verbs:
+      - get
+      - list
+      - watch
+
+---
```

Comment:
CVO folks want us to move away from having lists in manifests. Please move this to a separate file. If you keep it prefixed `03-rbac-...` it will run in the ordering I believe we want.

See https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/operators.md#names-of-manifest-files

Reply:
done